packages/uicore/src/frameworks/Tooltip/Tooltip.tsx

```diff
@@ -40,11 +40,20 @@ export const HarnessDocTooltip = ({
   const customWidth = widthValue ? Number(widthValue) : 400
   const tooltipContentHtml = _asHtml(asString)
 
+  const stopPropagation = (e: React.SyntheticEvent<HTMLElement, Event>) => {
+    e?.preventDefault()
+    e?.nativeEvent.stopImmediatePropagation()
+  }
+
   const tooltipJsxComponent = (
     <Popover
       popoverClassName={css.tooltipWrapper}
       position="auto"
       interactionKind={PopoverInteractionKind.HOVER}
+      onInteraction={(_, e) => {
+        e?.preventDefault()
```

Comment:
why are we doing both, is that required?

Reply:
To prevent the element's default behaviour and to stop the event from bubbling up the DOM, we need both events to fire.